cockroachcloud/cloud-org-sso.md

```diff
@@ -0,0 +1,126 @@
+
+    Keep the following in mind and communicate it to your members before enabling Cloud Organization SSO:
+
+      - If your organization includes members who are also members of other organizations, and this is the first of their organizations to enable Cloud Organization SSO, those members must be re-added to your organization. If they sign in using an authentication method with [autoprovisioning](#autoprovisioning) enabled, they are automatically added upon successful sign-in. **However, those who were previously admins must be granted the role again**.
+      - After you enable Cloud Organization SSO, all other members of your {{ site.data.products.db }} organization who were using [Basic SSO](cloud-org-sso.html#basic-sso) rather than an email and password must sign in again to regain access to your organization. After signing in, members retain the same access they had before the migration.
+
+During enablement of the feature, a list of affected members is shown, and those members are also notified individually.
+
+### Enforce a requirement to use SSO
+
+Using SSO with a centralized identity management and user provisioning is a recommended security practice. It helps to mitigate against the risk of unauthorized access to your organization's data and infrastructure by users who should no longer have access, such as when a {{ site.data.products.db }} account using password authentication is not deprovisioned when an employee leaves the organization.
+
+To help mitigate against this risk, SSO is preferred over password authentication, whether your organization uses Basic SSO or Cloud Organization SSO. If a member opts to log in using SSO, their default authentication method is automatically and their previous password is not retained. After a member switches to using SSO, they cannot switch back to using a password, but they can switch to a different SSO authentication method as long as the associated email address is identical. For a member to switch back to using a password, they must be removed and re-invited to your {{ site.data.products.db }} organization.
+
+Refer to [Require SSO](configure-cloud-org-sso.html#require-sso).
+
+## Frequently Asked Questions (FAQ)
+
+#### Will it work if I try to sign in with SSO without explicitly setting it up for my account, if I already use an email from an SSO Provider such as Gmail?
+
+Yes, as long as the email address associated with your SSO provider is exactly the same as the one associated with your existing {{ site.data.products.db }} account. After successfully signing in, you will be prompted to approve the updated authentication method for your account.
+
+To view your current authentication method, visit **My Account** in the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud).
+
+#### Once I change my active login method to a new SSO provider, can I still sign in using my email/password combination or my GitHub identity?
```

Comment:
```suggestion
#### Once a user changes their active login method to a new SSO provider, can they still sign in using an email/password combination or GitHub identity?
```

Reply:
Again, this question is relevant for Basic SSO and we should indicate that. For Cloud Org SSO, it'll depend on which methods are enabled and there's no explicit switch available (it's auto-deduced using the email address).